coalib/bears/Bear.py

```diff
@@ -24,6 +26,23 @@
 from .meta import bearclass
 
 
+def _setting_is_enabled(bear, key):
+    """
+    Check setting key is in section.
+
+    :param bear: Bear object.
+    :param key:  Setting key.
+    :return:     Setting object if setting key is in section else ``False``.
```

Comment:
I said this should return False, True or Setting .

You have a _setting_is_enabled which doesnt return True.  How nuts is that?